doc/source/whatsnew/v0.18.1.txt

```diff
@@ -131,6 +131,8 @@ These changes conform sparse handling to return the correct types and work to ma
 
 API changes
 ~~~~~~~~~~~
+- ``.swaplevel()`` for ``Series``, ``DataFrame``, ``Panel``, and ``MultiIndex`` now features defaults for its first two parameters ``i`` and ``j`` that swap the two innermost levels of the index.
```

Comment:
To make sure I understand: are you saying that (a) even though the subsequent bullet points are separated by blank lines, the blank line beneath this bullet point should disappear? or (b) that when I go in to the code to add the issue number, I should keep it on the same line and not let it land by accident on a subsequent line?